utils/utils.go

```diff
@@ -199,6 +210,22 @@ func pathToRegExp(localPath string) string {
 	return "^" + localPath + "$"
 }
 
+func antPathToRegExp(localPath string) string {
+	var SPECIAL_CHARS = []string{".", "^", "$", "+"}
+	for _, char := range SPECIAL_CHARS {
+		localPath = strings.Replace(localPath, char, "\\"+char, -1)
+	}
+	var wildcard = ".*"
+	localPath = strings.Replace(localPath, `?`, ".{1}", -1)
+	localPath = strings.Replace(localPath, `**/`, "#", -1)
```

Comment:
The pattern can include **#**, so let's find a way to avoid using it (or any other character) as a temporary replacement.